bin/raptor

```diff
@@ -82,6 +82,12 @@ var parent = parser
     help: 'path to store historical test metrics. Environment: RAPTOR_LOG',
     default: path.join(process.cwd(), 'raptor.log'),
     transform: utils.compose(utils.toAbsolute, utils.fromEnvironment('RAPTOR_LOG'))
+  })
+  .option('output', {
+    metavar: '<mode>',
+    help: 'output mode: normal or quiet. Environment: RAPTOR_OUTPUT',
+    default: 'normal',
```

Comment:
Actually, I think it should be easy now to go back to the idea of the boolean --quiet option.  Which solution do you prefer?